provisioner/roles/manage_ec2_instances/tasks/instances_security_automation.yml

```diff
@@ -1,168 +1,164 @@
     region: "{{ ec2_region }}"
-    owners: "{{ ec2_info[node2_node].owners }}"
+    owners: "{{ ec2_info['rhel7']['owners'] }}"
     filters:
-      name: "{{ ec2_info[ansible_node].filter }}"
-      architecture: "{{ ec2_info[node2_node].architecture }}"
+      name: "{{ ec2_info['rhel7']['filter'] }}"
+      architecture: "{{ ec2_info['rhel7']['architecture'] }}"
   register: amis
 
-- name: save ami for node2
+- name: save ami for snort
   set_fact:
-    node2_ami: >
-      {{ amis.images | selectattr('name', 'defined') | sort(attribute='creation_date') | last }}
+    snort_ami: >
+      {{ amis['images'] | selectattr('name', 'defined') | sort(attribute='creation_date') | last }}
 
-- name: Create EC2 instances for node2
+- name: Create EC2 instances for snort
   ec2:
     assign_public_ip: yes
     key_name: "{{ ec2_name_prefix }}-key"
     group: "{{ ec2_security_group }}"
-    instance_type: "{{ ec2_info[node2_node].size }}"
-    image: "{{ node2_ami.image_id }}"
+    instance_type: "{{ ec2_info['rhel7']['size'] }}"
+    image: "{{ snort_ami['image_id'] }}"
     region: "{{ ec2_region }}"
     exact_count: "{{ student_total }}"
     count_tag:
-      Workshop_node2: "{{ec2_name_prefix}}-node2"
+      Workshop_snort: "{{ec2_name_prefix}}-snort"
     wait: "{{ ec2_wait }}"
     vpc_subnet_id: "{{ ec2_vpc_subnet_id }}"
     volumes:
       - device_name: /dev/sda1
         volume_type: gp2
-        volume_size: "{{ ec2_info[node2_node].disk_space }}"
+        volume_size: "{{ ec2_info['rhel7']['disk_space'] }}"
         delete_on_termination: true
-  register: node2_output
+  register: snort_output
 
-- name: Ensure tags are present for node2
+- name: Ensure tags are present for snort
   ec2_tag:
     region: "{{ ec2_region }}"
-    resource: "{{item.1}}"
+    resource: "{{item[1]}}"
     state: present
     tags:
-      Name: "{{ ec2_name_prefix }}-student{{item.0 + 1}}-node2"
-      Workshop_node2: "{{ec2_name_prefix}}-node2"
+      Name: "{{ ec2_name_prefix }}-student{{item[0] + 1}}-snort"
+      Workshop_snort: "{{ec2_name_prefix}}-snort"
       Workshop: "{{ec2_name_prefix}}"
       Index: "{{ item[0] }}"
-      Student: "student{{item.0 + 1}}"
+      Student: "student{{item[0] + 1}}"
       AWS_USERNAME: "{{ linklight_user }}"
       Info: "AWS_USERNAME that provisioned this-> {{ linklight_user }}"
       Linklight: "This was provisioned through the linklight provisioner"
       Students: "{{student_total}}"
-      short_name: "node2"
+      short_name: "snort"
+      username: '{{ec2_info["rhel7"]["username"]}}'
   with_indexed_items:
-    - "{{ node2_output.instance_ids }}"
-  when: node2_output.instance_ids is not none
+    - "{{ snort_output['instance_ids'] }}"
+  when: snort_output['instance_ids'] is not none
 
-- name: find ami for node3
+- name: find ami for checkpoint
   ec2_ami_facts:
     region: "{{ ec2_region }}"
-    owners: "{{ ec2_info[node3_node].owners }}"
+    owners: "{{ ec2_info['checkpoint_firewall']['owners'] }}"
```

Comment:
See above - we first need to setup the management server, and then add the firewalls.